test/parser/parser-test.js

```diff
@@ -68,6 +59,15 @@ describe('The parser', function() {
         });
     });
 
+    // it('parses the simple-for program correctly', function(done) {
+    //     scan('./test/parser/inputs/valid/simple-for.yah', function(tokens) {
+    //         var program = parse(tokens);
+    //         program.toString().should.eql(expected_ast.simple_for);
+    //         error.count.should.be.eql(0);
+    //         done();
+    //     });
+    // });
+
```

Comment:
Don't comment tests out. Use `skip` instead. See https://mochajs.org/ and search for "skip".